cli/installer/docker_compose.go

```diff
@@ -106,12 +111,19 @@ func getDockerComposeFileContents(ui cliUI.UI, config configuration) []byte {
 		include = append(include, "cache", "queue", "demo-api", "demo-worker", "demo-rpc", "otel-collector")
 	}
 
-	filterContainers(ui, project, include)
+	// filter and update project
+	filterAndFixContainers(ui, project, include)
```

Comment:
Tech debt to the future: even changing the container to consider the demo projects, we didn't consider the initialization order of the containers.

Today Tracetest can start before `PokeshopAPI`, which can make our users try to run a test and receive an error because the API wasn't started yet.